src/app/components/common/shapes/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { IShapesLibraryPlugin } from "../../../../compiler/runtime/libraries/shapes";
+import { Shape } from "../../../shapes/Shape";
+import { Rectangle } from "../../../shapes/Rectangle";
+import { Ellipse } from "../../../shapes/Ellipse";
+import { Triangle } from "../../../shapes/Triangle";
+import { Line } from "../../../shapes/Line";
+import { TextShape } from "../../../shapes/Text";
+
+export class ShapesComponent /*extends React.Component<Props, State>*/ implements IShapesLibraryPlugin {
```

Comment:
Remove the commented out part, and I would rename this file to ShapesLibraryPlugin, and move this out of the /components director to the /shapes directory (and delete this components/common/shapes directory)